platformLibs/src/platform/mingw/posix.def

```diff
@@ -1,11 +1,114 @@
 package = platform.posix
-headers = assert.h complex.h ctype.h dirent.h errno.h fcntl.h \
-    fenv.h float.h ftw.h getopt.h inttypes.h libgen.h limits.h \
-    locale.h math.h memory.h pthread.h sched.h search.h semaphore.h \
-    setjmp.h signal.h stdint.h stdio.h stdlib.h string.h \
-    time.h uchar.h unistd.h utime.h wchar.h wctype.h
+
```

Comment:
We likely need matching compiler options with other headers.

Reply:
You mean same as in `windows.def`?
```
compilerOpts = \
    -DUNICODE \
    -DWINVER=0x0601 \
    -D_WIN32_WINNT=0x0601 \
    -DWINAPI_FAMILY=3 \
    -DOEMRESOURCE \
    -Wno-incompatible-pointer-types \
    -Wno-deprecated-declarations
```
Tried it and recompile - almost no change in `posix.kt`
``` diff
diff -ur a/posix.kt b/posix.kt
--- a/posix.kt	2019-10-30 12:58:39.175066200 +0500
+++ b/posix.kt	2019-10-30 13:48:24.090793700 +0500
@@ -6512,8 +6512,6 @@
 
 const val __MSVCRT_VERSION__: Int = 1792
 
-const val _WIN32_WINNT: Int = 1537
-
 const val MINGW_HAS_SECURE_API: Int = 1
 
 const val __STDC_SECURE_LIB__: Int = 200411
@@ -8062,9 +8060,9 @@
 
 const val CLOCK_THREAD_CPUTIME_ID: Int = 3
 
-const val _INC_CRT_UNICODE_MACROS: Int = 2
+const val _INC_CRT_UNICODE_MACROS: Int = 1
 
-val __MINGW_PROCNAMEEXT_AW: String get() = "A"
+val __MINGW_PROCNAMEEXT_AW: String get() = "W"
 
 const val INVALID_SOCKET: SOCKET = 18446744073709551615u
 
@@ -8314,7 +8312,7 @@
 
 const val SO_PROTOCOL_INFOW: Int = 8197
 
-const val SO_PROTOCOL_INFO: Int = 8196
+const val SO_PROTOCOL_INFO: Int = 8197
 
 const val PVD_CONFIG: Int = 12289
 
```